src/cli/commands/color_commands.rs

```diff
@@ -77,6 +78,37 @@ color_command!(MixCommand, config, matches, color, {
     mix(&base, &color, fraction)
 });
 
+color_command!(ColorblindCommand, config, matches, color, {
+    // The type of colorblindness selected (protanopia, deuteranopia, tritanopia)
+    let cb_ty = matches.value_of("type").expect("required argument");
+    let cb_ty = cb_ty.to_lowercase();
+
+    // Coefficients here are taken from
+    // https://ixora.io/projects/colorblindness/color-blindness-simulation-research/
```

Comment:
I think it would be great to move this part to the `pastel` library. Maybe be adding a `pub enum ColorblindnessType { Protanopia, … }` and a function `Color::simulate_colorblindness(type: ColorblindnessType)` that would perform the calculations below.

This way, we could later use this functionality in other commands as well. One example from the top of my head: we could add a `--colorblindness <type>` option (or similar) to the `pastel distinct` command and it could call `simulate_colorblindness(type)` before computing the perceived color difference. This way, we could (hopefully?) create visually distinct colors for a given type of color blindness.

Reply:
Also good point. Will do. As for distance, assuming non-colorblind people see the post-processed colors the same way a colorblind person does (which I believe is true), then any standard distance metric will continue to apply.